gui_views/attribute_view.py

```diff
@@ -71,7 +71,14 @@ def modifyParam(self, currentAttribute, updatedValue):
                     catalog["SimObject"][name]['ports'][currentAttribute]['Type']
         else:
             instance_params[currentAttribute]["Value"] = updatedValue
+
         self.state.addToHistory()
+        self.state.highlightIncomplete()
+
+        # remove highlight for object if it is complete now
+        object = self.state.selected_sym_objects[0]
+        if not object.incomplete:
+            object.rect.setBrush(QColor("White"))
```

Comment:
should it change to green since its in the selected_sym_objects list?

Reply:
Good point, just changed it to green